src/resolvers/donationResolver.ts

```diff
@@ -94,30 +96,43 @@ export class DonationResolver {
     @Ctx() ctx: MyContext,
     @Arg('skip', { defaultValue: 0 }) skip: number,
     @Arg('take', { defaultValue: 10 }) take: number,
+    @Arg('traceable', { defaultValue: false }) traceable: boolean,
     @Arg('projectId', type => Number) projectId: number,
   ) {
-    const query = this.donationRepository
-      .createQueryBuilder('donation')
-      .leftJoinAndSelect('donation.user', 'user')
-      .where(`donation.projectId = ${projectId}`);
-
-    const [donations, donationsCount] = await query
-      .take(take)
-      .skip(skip)
-      .getManyAndCount();
-    const balance = await query
-      .select('SUM(donation.valueUsd)', 'usdBalance')
-      .getRawOne();
-    const ethBalance = await query
-      .select('SUM(donation.valueEth)', 'ethBalance')
-      .getRawOne();
-
-    return {
-      donations,
-      totalCount: donationsCount,
-      totalUsdBalance: balance.usdBalance,
-      totalEthBalance: ethBalance.ethBalance,
-    };
+    const project = await Project.findOne({
+      id: projectId,
+    });
+    if (!project) {
+      throw new Error(errorMessages.PROJECT_NOT_FOUND);
+    }
+
+    if (traceable) {
+      const { total, donations } = await getCampaignDonations({
+        campaignId: project.traceCampaignId as string,
+        take,
+        skip,
+      });
+      return {
+        donations,
+        totalCount: total,
+        totalUsdBalance: project.totalTraceDonations,
+      };
+    } else {
+      const query = this.donationRepository
+        .createQueryBuilder('donation')
+        .leftJoinAndSelect('donation.user', 'user')
+        .where(`donation.projectId = ${projectId}`);
+
+      const [donations, donationsCount] = await query
+        .take(take)
+        .skip(skip)
+        .getManyAndCount();
+      return {
+        donations,
+        totalCount: donationsCount,
+        totalUsdBalance: project.totalDonations,
```

Comment:
Mateo told me to add the totalEth balance for the frontend / new design that's why I had it. We should add it to the database and update our usdvalue updater script to include totalEth value. 
But I am not sure how this will interact with the trace, does trace have ethvalue too?

Reply:
@mateodaza  @CarlosQ96 
We dont have `ethValue` for trace donations, is it ok to send it just for givethio donations?

Reply:
As per the meeting we can ignore the totalEthValue and merge.